aws/aws-event-streams/src/main/java/software/amazon/smithy/java/aws/events/RpcEventStreamResponse.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.java.aws.events;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Flow;
+import software.amazon.smithy.java.core.schema.SerializableStruct;
+import software.amazon.smithy.java.core.serde.event.EventDecoderFactory;
+import software.amazon.smithy.java.core.serde.event.EventStreamFrameDecodingProcessor;
+import software.amazon.smithy.java.http.api.HttpResponse;
+import software.amazon.smithy.java.io.datastream.DataStream;
+
+/**
+ * A util class to deserialize event streams responses for RPC protocols.
+ */
+public final class RpcEventStreamResponse {
```

Comment:
same thing here. Maybe even combine this utils class with the request utils class.

Reply:
Sure.